### PR TITLE
Provide PHP 8.2 support

### PR DESCRIPTION
**Pull request by raviks789**
Blocked by #5048 

Depends on Icinga/icinga-php-thirdparty#11

### CODE REVIEW

**Review by lippserd — CHANGES_REQUESTED** [review]

File: library/Icinga/Util/LessParser.php [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 class LessParser extends lessc
 {
+    public $importDisabled;
```
[the remark this answers is not in the document]
Comment (raviks789) [reply library/Icinga/Util/LessParser.php:11]: I did not find this property in `lessc` and if I do not introduce this property here I encountered `PHP Deprecated:  Creation of dynamic property Icinga\Util\LessParser::$importDisabled is deprecated in /icingaweb2/library/Icinga/Web/LessCompiler.php on line 66` error.

[the remark this answers is not in the document]
Comment (lippserd) [reply library/Icinga/Util/LessParser.php:11]: So they removed the functionality to set import to disabled. Remove that property and the code setting it to false.

**Review by lippserd — APPROVED** [review]

**Comment by lippserd** [comment]
@raviks789 Please rebase.

**Review by lippserd — CHANGES_REQUESTED** [review]
localhost_icingaweb2_layout_menu

**Comment by lippserd** [comment]
> localhost_icingaweb2_layout_menu

@raviks789 @sukhwinder33445 Is this detected/fixed somewhere by PHPStan, etc?

**Comment by raviks789** [comment]
> > localhost_icingaweb2_layout_menu
> 
> @raviks789 @sukhwinder33445 Is this detected/fixed somewhere by PHPStan, etc?

The dynamic properties seems to be have been identified in #5040 by PHPStan. A few of them have been fixed by  @sukhwinder33445 in the same PR. Should I move those related commits to this PR?

**Comment by sukhwinder33445** [comment]
> > localhost_icingaweb2_layout_menu
> 
> @raviks789 @sukhwinder33445 Is this detected/fixed somewhere by PHPStan, etc?

Yes, detected and fixed [here](https://github.com/Icinga/icingaweb2/pull/5040/commits/e4a02f913020f09284775bce217981e854863d78#diff-beff3749684c8fae946eb0c0411b44a95af7d21e7114b92f01f4db9f350ba6d6).

**Comment by sukhwinder33445** [comment]
Screenshot 2023-07-19 at 15 54 43


## Reproduce

Call `null->getMyBug();` in any controller action and visit the url.

**Pull request closed without merging**